src/js/components/Books/Books.js

```diff
@@ -14,10 +14,12 @@ class  Books extends Component {
         <br/>
         <div className="container ">
             <div className="row">
-                <div className="col-4  noPaddMar"><Bookcard/></div>           
-                <div className="col-4  noPaddMar"><Bookcard/></div>           
-                <div className="col-4  noPaddMar"><Bookcard/></div>           
-                <div className="col-4  noPaddMar"><Bookcard/></div>           
+                <div className="col-4 col-sm-3 col-md-2 noPaddMar"><Bookcard/></div>           
```

Comment:
There is an incredible amount of repetition here.
Use a for loop for dynamic datas

Reply:
Thank you i fixed this